mountpoint-s3/scripts/fio/write_latency/seq_write_small.fio

```diff
@@ -0,0 +1,14 @@
+[global]
+name=fs_bench
+bs=1B
+
+[sequential_write_small_file]
```

Comment:
It's not really clear from the name what number we want to measure here. Could we have better naming for it?